kyuubi-common/src/main/scala/org/apache/kyuubi/operation/OperationStatus.scala

```diff
@@ -27,4 +27,5 @@ case class OperationStatus(
     lastModified: Long,
     completed: Long,
     hasResultSet: Boolean,
-    exception: Option[KyuubiSQLException] = None)
+    exception: Option[KyuubiSQLException] = None,
+    jobProgressUpdate: Option[JobProgressUpdate] = None)
```

Comment:
may be jobProgressUpdate: Option[TProgressUpdateResp] is OK to use

Reply:
> may be jobProgressUpdate: Option[TProgressUpdateResp] is OK to use

Does this mean I need to remove the JobProgressUpdate class?

Reply:
yes, it seem that we don't need wrap/unwrap at engine and server side

Reply:
> yes, it seem that we don't need wrap/unwrap at engine and server side

Thanks, I will remove it later.